src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1285,7 +1320,7 @@ protected function {$methodNameAlias}()
         return $code;
     }
 
-    private function addInlineRequires(): string
+    private function addInlineRequires(?array &$preload): string
```

Comment:
should `null` be allowed ?

Reply:
to allow calling the method with an uninitialized variable